src/test/java/org/prebid/server/bidder/invibes/InvibesBidderTest.java

```diff
@@ -120,31 +127,84 @@ public void makeHttpRequestsShouldReturnErrorWhenSiteIsNotPresent() {
     }
 
     @Test
-    public void makeHttpRequestsShouldReturnErrorWhenSiteIsNotPresent2() {
+    public void shouldCreateRequestWithDataFromEveryImpression() throws JsonProcessingException {
+        // given
+        final List<Imp> imps = Arrays.asList(givenImp(
+                impBuilder -> impBuilder
+                        .banner(Banner.builder().h(10).w(11).build()), ExtImpInvibes.of("12", 15,
+                        InvibesDebug.of("test1", true))),
+                givenImp(impBuilder -> impBuilder
+                        .banner(Banner.builder().h(14).w(15).build()), ExtImpInvibes.of("15", 1001,
+                        InvibesDebug.of("test2", false)))
+        );
+        final BidRequest bidRequest = BidRequest.builder()
+                .site(Site.builder().page("www.awesome-page.com").build())
+                .imp(imps)
+                .build();
+        // when
+        final Result<List<HttpRequest<InvibesBidRequest>>> result = invibesBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1)
+                .extracting(HttpRequest::getPayload).hasSize(1);
+        InvibesBidParams expectedBidParams =
+                mapper.readValue(result.getValue().get(0).getPayload().getBidParamsJson(), InvibesBidParams.class);
+        assertThat(expectedBidParams.getPlacementIds()).hasSize(2)
+                .containsOnly("12", "15");
+        final Format firstExpectedFormat = Format.builder().w(11).h(10).build();
+        final Format secondExpectedFormat = Format.builder().w(15).h(14).build();
+        assertThat(expectedBidParams.getProperties().values()).hasSize(2)
+                .containsOnly(InvibesPlacementProperty.builder()
+                                .formats(Collections.singletonList(firstExpectedFormat))
+                                .build(),
+                        InvibesPlacementProperty.builder()
+                                .formats(Collections.singletonList(secondExpectedFormat))
+                                .build());
+    }
```

Comment:
Ok. 
1. Make variable for values that need to be checked. For example. What is 11. int firstImpBannerW = 10
firstImpBannerH = 11
etc

2. Usage of `extracting(HttpRequest::getPayload).hasSize(1)` just to extract this value next is not necessary.
Try to make clear pipeline.
smth like this
```        assertThat(result.getValue())
                .extracting(HttpRequest::getPayload)
                .extracting(invibesBidRequest -> mapper.readValue(invibesBidRequest.getBidParamsJson(), InvibesBidParams.class))
                .containsOnly(expectedInivibesBidParameters)
```
Like in the method below